android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -129,16 +129,25 @@ protected void doInBackgroundGuarded(Void... params) {
       File source = new File(mUri.getPath());
       FileChannel input = null, output = null;
       try {
-        File environment;
-        if ("mov".equals(mOptions.getString("type"))) {
-          environment = Environment.getExternalStoragePublicDirectory(
-                  Environment.DIRECTORY_MOVIES);
+        boolean isAlbumPresent = !"".equals(mOptions.getString("album"));
+        
+        final File environment;
+        // Media is not saved into an album when using Environment.DIRECTORY_DCIM.
+        if (isAlbumPresent) {
+            if ("video".equals(mOptions.getString("type"))) {
```

Comment:
The formatting for Java files is 2 spaces in the rest of the code base

Reply:
@bartolkaruza should be fixed now, I didn't noticed it!